### DISCUSSION TITLE
CON-0001

### DISCUSSION THREAD

**Discussion by AnrivaDeloitte** [OP-TED/eForms-SDK#938] [Q&A] [answered]
What does "CON-000X" stand for? When and in what cases should it be filled?

![image](https://github.com/OP-TED/eForms-SDK/assets/144999812/0aa2dcd2-7a44-40f4-ba5c-dfc3299063d7)

**Comment by YvesJo** [comment, marked as answer]
Hi,
Any contract has an associated unique Technical Identifier in the form "_CON-xxxx_" (OPT-316-Contract). 
This technical ID may only exist when a contract exist.
Details about a contract are to be provided as soon as a contract exists.
